docs/README.md

```diff
@@ -2,6 +2,10 @@
 
 Welcome to the documentation of the **Cosmos Hub application: `gaia`**.
 
+## What is Gaia?
+
+- [Gaia](./what-is-gaia.md)
```

Comment:
Where is this located? I don't see it in docs?

Reply:
yes, this would be in alessio's branch alessio/docs